boards/riscv/longan_nano/support/openocd.cfg

```diff
@@ -1,17 +1,25 @@
 # Copyright (c) 2021 Tokita, Hiroshi <tokita.hiroshi@gmail.com>
 # SPDX-License-Identifier: Apache-2.0
 
-interface ftdi
+#interface ftdi
```

Comment:
Why do we keep commented-out lines in this file?

Reply:
@fkokosinski 
I am very sorry.
I accidentally included the patch I had for my environment.
Removed it.